README.md

```diff
@@ -84,36 +99,43 @@ For certain versions, multiple patches may be available. It's important to ensur
 
 1. Go to "korrekt"
 
-2. Run `cargo run` with relevant halo2 version feature flag (the circuit is hard-coded in main.rs, in the future this should be a library)
-    You must enable at least one of the available feature flags
+2. Run `cargo run` for the default feature falg (`use_zcash_halo2_proofs`). 
+
+    ```bash
+    cargo run
+    ```
+
+    For other versions of halo2 run `cargo run` with relevant halo2 version feature flag:
 
     ```bash
-    cargo run --no-default-features --features use_zcash_halo2_proofs
     cargo run --no-default-features --features use_pse_halo2_proofs
     cargo run --no-default-features --features use_axiom_halo2_proofs
     cargo run --no-default-features --features use_scroll_halo2_proofs
     cargo run --no-default-features --features use_pse_v1_halo2_proofs
     ```
 
-    `--no-default-features` for the current setup where the default flag is set to `use_zcash_halo2_proofs`.
+   A circuit is hard-coded in main.rs, in the future this should be a library.
 
 ## How to test
 
 1. Go to "korrekt"
 
-2. Run `cargo test -- --test-threads=1`  with relevant halo2 version feature flag.
-    You must enable at least one of the available feature flags
+2. Run `cargo test -- --test-threads=1` for the default feature falg (`use_zcash_halo2_proofs`).
 
     ```bash
-    cargo test --no-default-features --features use_zcash_halo2_proofs -- --test-threads=1
+    cargo test -- --test-threads=1
```

Comment:
Instead of passing `--test-threads=1` explicitly in the command-line, it can also be made default by specifying it in the file `.cargo/config.toml`.

Reply:
Good point, I added the config and updated the readme.